pkg/deploy/generator/templates_rp.go

```diff
@@ -297,14 +298,17 @@ func (g *generator) rpPredeployTemplate() *arm.Template {
 			"extraServiceKeyvaultAccessPolicies":
 			p.Type = "array"
 			p.DefaultValue = []interface{}{}
+		case "rpNsgPortalSourceAddressPrefixes":
+			p.Type = "array"
+			p.DefaultValue = []string{}
 		case "keyvaultPrefix":
 			p.MaxLength = 24 - max(len(env.ClusterKeyvaultSuffix), len(env.ServiceKeyvaultSuffix), len(env.PortalKeyvaultSuffix))
 		}
 		t.Parameters[param] = p
 	}
 
+	t.Resources = append(t.Resources, g.rpSecurityGroup()...)
 	t.Resources = append(t.Resources,
-		g.rpSecurityGroup(),
```

Comment:
This removal would be reverted and a new one would be added for the new function `g.rpSecurityGroupFairFaxAdminPortal()`.